### PR TITLE
Wp add llama3 experiments

### PR DESCRIPTION
**Pull request by willpoulett**
# Summary

**What/Why**

- Added llama3 template in config
- Added llama3 as the example full_pipeline_example.ipynb
- Added llama3 to experimental_pipeline_example.ipynb. This still contains llama2, so demonstrates an example of combine_overrides
- Added the outputs of these to the data folder

**Your approach**

LLama3 is a more recent model so it is appropriate to have it in the examples.

# Checklists

This pull/merge request meets the following requirements:

- [Y] code runs
- [Y] [developments are ethical][data-ethics-framework]
- [Y] developments are secure (no data or notebook output)
- [Y] you have made proportionate checks that the code works correctly
- [ Y] [minimum usable documentation][agilemodeling] written in the `docs` folder

Comments have been added below around the incomplete checks.

[agilemodeling]: http://agilemodeling.com/essays/documentLate.htm
[data-ethics-framework]: https://www.gov.uk/government/publications/data-ethics-framework

### CODE REVIEW

File: notebooks/experimental_pipeline_example.ipynb
Comment (scarlett-k-nhs) [inline notebooks/experimental_pipeline_example.ipynb:1]: Minor thing can we bold UniversalNER so it matches the same style as llama2 and 3

**Comment by scarlett-k-nhs** [comment]
Another thing if you have changed the experiment folder can you delete the previous folder as we now don't have the code it was run on so we might as well delete it.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,14 @@ In **./notebooks/generative_module** there is a set of notebooks exploring how t
 
 Note that a seed has not been implemented to reproduce the outputs shown.
 
+###  License Agreements
+
```
Comment (scarlett-k-nhs) [inline README.md:93]: This is good to let the user know how to sign up but I am wondering if we need to make it a little more formal:

## Licensing

This project uses LLaMA 2 and LLaMA 3 models from Meta AI. Usage of these models is governed by the non-commercial license provided by Meta. 

### LLaMA 2

LLaMA 2 is licensed under Meta's non-commercial license. For more details, please refer to [Meta's LLaMA 2 licensing page](https://example.com/llama2-license). To sign the corresponding license agreement you can apply for this via huggingface[https://huggingface.co/meta-llama/Llama-2-70b].

### LLaMA 3

LLaMA 3 is also licensed under Meta's non-commercial license. For more details, please refer to [Meta's LLaMA 3 licensing page](https://example.com/llama3-license). To sign the corresponding license agreement you can apply for this via huggingface[https://huggingface.co/meta-llama/Meta-Llama-3-70B].

By using these models in this project, you agree to comply with the licensing terms provided by Meta.

For any commercial use or further inquiries, please contact Meta AI directly.

File: README.md [hunk 2]
```diff
@@ -89,6 +89,14 @@ In **./notebooks/generative_module** there is a set of notebooks exploring how t
...
+If using Llama models from META, please read and sign the corresponding licence agreements.
+They can be found at the top of the model card on the huggingface website:
+
+- [Llama-3-70B](https://huggingface.co/meta-llama/Meta-Llama-3-70B)
+- [Llama-2-70B](ttps://huggingface.co/meta-llama/Llama-2-70b)
```
Comment (scarlett-k-nhs) [inline README.md:98]: https?

File: README.md [hunk 3]
```diff
@@ -89,6 +89,14 @@ In **./notebooks/generative_module** there is a set of notebooks exploring how t
 
 Note that a seed has not been implemented to reproduce the outputs shown.
 
+###  License Agreements
+
+If using Llama models from META, please read and sign the corresponding licence agreements.
```
Comment (scarlett-k-nhs) [inline README.md:94]: license

**Review by danjscho — APPROVED** [review]

**Pull request merged**